src/com/theironyard/novauc/ATM2.java

```diff
@@ -0,0 +1,93 @@
+package com.theironyard.novauc;
+
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class ATM2 {
+// I had to create a public static Scanner called scannerDouble = new Scanner(System.in); or else I would have had issues
+    public static Scanner scanner = new Scanner(System.in);
+    public static Scanner scannerDouble = new Scanner(System.in);
+    public static Customer customer = new Customer();
+//    public static Customers customers = new Customers();
+
+//    public static Person person = new Person();
+// I did not need this public static Person above; but I did need to go and actually create a Person "Class" and then
+    //go in to that tab and actually create "getters and setters" using Code - Generate - Getters and Setters
+    public static void main(String[] args) throws Exception {
+
+        System.out.println("Welcome, customer");
+
+
+        customer.chooseName();
+        customer.chooseRemove();
+        customer.chooseDesire();
+        customer.chooseResponse();
+
+
+        customer.doAction(1);
+        customer.chooseResponse();
+
```

Comment:
The way this is setup up you eventually get to a point where the user can only make a withdraw.  You should call your chooseResponse and chooseRemove methods inside of the chooseDesire method.  And then call chooseDesire every time.  If that is not clear, talk to me about it in class.